jekyll/package.json

```
@@ -1,6 +1,6 @@
 {
   "scripts": {
-    "install": "yum install ruby23-devel.x86_64 && gem install jekyll bundler --no-ri --no-rdoc && bundle install",
-    "build": "jekyll build && mv _site public"
+    "now-install": "yum install ruby23-devel.x86_64 && gem install bundler --no-ri --no-rdoc && bundle install",
```

I thought we could use `now-install` but this will need to be `install`

```suggestion
    "install": "yum install ruby23-devel.x86_64 && gem install bundler --no-ri --no-rdoc && bundle install",
```